backend/routes/issues.js

```diff
@@ -8,14 +8,17 @@ router.route('/').post(async (req, res) => {
   const title = req.body.title
   const description = req.body.description
   const labels = req.body.labels
+  const date = Date.parse(req.body.date)
```

Comment:
Date fields are not needed to be provided by client. You can set the creationDate to current date. And modificationDate can be updated when any update is done on the issue.